c10/hammerblade/CMakeLists.txt

```diff
@@ -75,12 +75,12 @@ else()
   # Kernel binary
```

Comment:
Emmm this file shouldn't be touched. I'll help you revert the changes. Dont worry about it